Destination/Destination.php

```diff
@@ -3,15 +3,25 @@
 namespace Zenstruck\BackupBundle\Destination;
 
 use Psr\Log\LoggerInterface;
+use \Countable;
+use \IteratorAggregate;
 
 /**
  * @author Kevin Bond <kevinbond@gmail.com>
  */
-interface Destination
+interface Destination extends Countable, IteratorAggregate
```

Comment:
Ok, I see what you mean.

If you are building a GUI, do you not want the functionality to remove backups as well? We should think about what we want the GUI to do. That functionality might be out the scope of the `Destination`. I intended the destination to only push a file.

For instance, the S3CmdDestination can really never do more than push - it can't list or remove (well i suppose it could but it would be exceedingly difficult).

I think we need a separate system for "administrating" the backups. Maybe even a separate bundle.